### PR TITLE
Import code

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: manifests/02_crd.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+              format: int64
+            readyReplicas:
+              description:
+                readyReplicas indicates how many replicas are ready and
+                at the desired state
+              type: integer
+              format: int32
+            version:
+              description: version is the level this availability applies to
+              type: string
```
Comment (gnufied) [inline manifests/02_crd.yaml]: Not a blocker but we will have to split the manifests that OLM will be automatically installing (such as CRD) vs manifests that will be synced by the operator.

**Comment by gnufied** [comment]
lets merge this and then fix up in follow up commits.

File: manifests/02_crd.yaml (hunk 1 above)
Comment (bertinatto) [reply manifests/02_crd.yaml]: Yeah, these files are here just to make it easier to install the operator manually, they don't work with OLM.

**Pull request merged**